prometheus/rootfs/bin/boot.sh

```diff
@@ -22,18 +22,23 @@ done
 
 set +e
 
-OPTS=""
+until [ -n "${DEIS_MONITOR_ALERT_SERVICE_HOST+1}" ]; do
```

Comment:
K8S configures the resolve to "dial down" the DNS tree:

```
/ # cat /etc/resolv.conf
nameserver 10.3.0.10
nameserver 10.0.0.2
search default.svc.cluster.local svc.cluster.local cluster.local us-west-2.compute.internal
options ndots:5
```

Where 'default' really represents the "namespace your Pod is currently running in". By convention, we should dial the hostname without any qualification so you would get the service in your current namespace like `foo-service`. If you need to reach out to another namespace, you could prefix that... `foo-service.foo-namespace.svc.cluster.local`.